.github/workflows/auto-build.yml

```diff
@@ -30,8 +30,10 @@ jobs:
           sudo tar -xvf sealer.tar.gz -C /usr/bin
           sudo sealer login ${{secrets.PRIVATEWAREHOUSE}}
           sudo sed -i "s/v1.20.14/$version/g" ./rootfs/etc/kubeadm.yml
+          sudo sed -i "s/v1.20.14/$version/g" ./rootfs/Metadata
```

Comment:
@kakaZhou719 metadata already has version feild?

Reply:
no, here we need to write the value of "version" field.  @Stevent-fei .i mean we write a json file with content like `{
  "version": "v1.19.8",
  "arch": "amd64"
}
`